app/src/main/java/com/untitled/timespace/ui/home/HomeFragment.kt

```diff
@@ -1,35 +1,30 @@
 package com.untitled.timespace.ui.home
 
-import androidx.lifecycle.observe
-import androidx.navigation.NavArgs
-import androidx.navigation.fragment.navArgs
-import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import androidx.navigation.ui.setupWithNavController
 import com.untitled.timespace.R
 import com.untitled.timespace.base.BaseFragment
 import com.untitled.timespace.databinding.FragmentHomeBinding
-import javax.inject.Inject
 
 internal class HomeFragment : BaseFragment<FragmentHomeBinding, HomeViewModel>() {
 
     override val layoutRes = R.layout.fragment_home
     override val viewModelClass = HomeViewModel::class
-    override val navArgs: NavArgs by navArgs<HomeFragmentArgs>()
 
-    @Inject
-    lateinit var homeAdapter: HomeAdapter
+    private val childNavController: NavController
+        get() {
```

Comment:
```suggestion
        get() = Navigation.findNavController(requireView().findViewById(R.id.navigation_home_fragment))
```
이렇게 한줄일때는 이런식으로 코딩 스타일 맞추는게 좋지않을까요?